src/test/java/uk/gov/digital/ho/pttg/api/HttpResponse.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.digital.ho.pttg.api;
+
+public class HttpResponse {
```

Comment:
If you're using the apache http library is there any reason not to use their HttpResponse?

Reply:
I just thought it was a bit clunky and low-level in its implementation. I didn't want to have to write code like
```
int responseCode = response.getStatusLine().getStatusCode();
String responseBody = EntityUtils.toString(response.getEntity())
```
in the tests themselves. This simplified view allows me to push the response reading into the Client.

Reply:
Ah ok, you're isolating the Apache domain - makes sense.